posthog.go

```diff
@@ -278,12 +301,17 @@ func (c *client) GetFeatureFlag(flagConfig FeatureFlagPayload) (interface{}, err
 		return false, err
 	}
 
-	if c.featureFlagsPoller == nil {
-		errorMessage := "specifying a PersonalApiKey is required for using feature flags"
-		c.Errorf(errorMessage)
-		return "false", errors.New(errorMessage)
+	var flagValue interface{}
+	var err error
+
+	if c.featureFlagsPoller != nil {
+		// get feature flags from the poller, which uses the personal api key
+		flagValue, err = c.featureFlagsPoller.GetFeatureFlag(flagConfig)
+	} else {
```

Comment:
should I log that we're falling back here?  Does it matter?

Reply:
maybe, but only at a trace/info/debug level. Might be good for investigations.